File: java/estruturas-repetitivas/ParesConsecutivos.java

import java.util.Scanner;

public class ParesConsecutivos {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int x, soma, cont, atual;

        System.out.print("Digite um numero inteiro: ");
        x = sc.nextInt();

        while (x > 0) {
            if (x % 2 == 1) {
                x++;
            }

            soma = 0;
            atual = x;
            cont = 0;

            while(cont < 5) {
                soma = soma + atual;
                atual += 2;
                cont++;
            }

            System.out.printf("SOMA = %d %n", soma);

            System.out.print("Digite um numero inteiro: ");
            x = sc.nextInt();
        }

        sc.close();
    }
}

File: js/estruturas-repetitivas/paresConsecutivos.js
const readline = require('readline-sync');

let X = Number(readline.question("Digite um numero: "));

while (X > 0) {
    if (X % 2 === 1) {
        X++
    }

    let soma = 0;
    let atual = X;
    let i = 0;

    while (i < 5) {
        soma += atual;
        atual += 2;
        i++;
    }
    console.log(`SOMA = ${soma}`);

    X = Number(readline.question("Digite um numero: "));
}